Could not find libz3.so; consider adding the directory containing it to
  - your system's PATH environment variable,
  - the Z3_LIBRARY_PATH environment variable, or 
  - to the custom Z3_LIB_DIRS Python-builtin before importing the z3 module, e.g. via
    import builtins
    builtins.Z3_LIB_DIRS = [ '/path/to/z3/lib/dir' ] # directory containing libz3.so
[OSError('libstdc++.so.6: cannot open shared object file: No such file or directory'), TypeError('expected str, bytes or os.PathLike object, not NoneType'), OSError('libz3.so: cannot open shared object file: No such file or directory')]
